### PR TITLE
Dala 5 landing page

### PR DESCRIPTION
**Pull request by stewal19**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
  - [x] The new feature was observed "in running software"
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to preview using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on preview
- [x] There is at least one picture for each story, which was created before coding has started

### CODE REVIEW

File: dataland-frontend/src/components/pages/CompanyTaxonomySample.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    TheContent,
+    TheHeader, BackButton, MarginWrapper, EuTaxoSearchBar, TaxonomyData, CompanyInformation, Button
+  },
+  data: () => ({
+      companyIdList: []
+  }),
+  props: {
+    companyIndex: {
+      type: Number,
+      default: 1
```
Comment (SiegfriedSobkowiak) [inline dataland-frontend/src/components/pages/CompanyTaxonomySample.vue:63]: Verstehe ich das richtig das wir einfach alle companies als Liste uns holen und dann das erste Element anzeigen weil wir erwarten das es Adidas ist? Warum suchen wir nicht direkt nach Adidas?

File: docker-compose.yml [hunk 2]
```diff
@@ -69,7 +69,7 @@ services:
   edc-dummyserver:
     restart: always
     profiles: ["testing", "development"]
-    image: ghcr.io/d-fine/datalandedc/edcdummyserver:latest
+    image: ghcr.io/d-fine/datalandedc/edcdummyserver:RELEASE-0.0.4
```
[the remark this answers is not in the document]
Comment (stewal19) [reply docker-compose.yml:72]: Bin ich dran.

[the remark this answers is not in the document]
Comment (stewal19) [reply docker-compose.yml]: Sind angepasst worden

File: settings.gradle.kts [hunk 3]
```diff
@@ -21,7 +21,7 @@ dependencyResolutionManagement {
             library("rs-api", "jakarta.ws.rs:jakarta.ws.rs-api:3.1.0")
 
             library("skyminder-client", "org.dataland:skyminder-client:0.1.3")
-            library("dataland-edc-client", "org.dataland:dataland-edc-client:0.0.2")
+            library("dataland-edc-client", "org.dataland:dataland-edc-client:0.0.4")
```
[the remark this answers is not in the document]
Comment (stewal19) [reply settings.gradle.kts]: Ist angepasst.

**Pull request merged**